### PR TITLE
[Master] Update Badges and logo on Readme

### PR DESCRIPTION
**Pull request by dkijania**
Update build badges and logo

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by ymekuria — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Review by MartinMinkov — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

File: README.md [hunk 1]
```diff
@@ -60,3 +64,7 @@ The [Node Developers](https://docs.minaprotocol.com/node-developers) docs contai
 [Apache 2.0](LICENSE)
 
 Commits older than 2018-10-03 do not have a [LICENSE](LICENSE) file or this notice, but are distributed under the same terms.
+
+[^1]: Develop is a mainline branch containig code that may be not compatible with current mainnet and may require major upgrade (hardfork).
```
Comment (mrmr1993) [inline README.md:68]: Nit: `containing`

File: README.md [hunk 2]
```diff
@@ -60,3 +64,7 @@ The [Node Developers](https://docs.minaprotocol.com/node-developers) docs contai
...
+[^2]: Compatible is a mainline branch containig code which does not need hardfork in order to apply it to mainnet.
```
Comment (mrmr1993) [inline README.md:69]: Nit: `containing`

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**